crates/core/src/globals.rs

```diff
@@ -0,0 +1,249 @@
+use anyhow::anyhow;
+use quickjs_wasm_rs::{Context, Value};
+use quickjs_wasm_sys::{
+    ext_js_exception, ext_js_undefined, size_t as JS_size_t, JSContext, JSValue, JS_FreeCString,
+    JS_ToCStringLen2,
+};
```

Comment:
Can we add a `TODO` or `FIXME` to indicate that this patter should be fixed and should not be adopted?

Reply:
Alternatively, one "easier" fix would be to re-export the sys types from the core crate. I think that generally I prefer that as a short term measure.

Reply:
So I added the deprecated attribute but I couldn't get the warnings to actually show up. But I don't to spend too much time looking into that if I could spend the equivalent time on rewriting that code to use `wrap_callback` instead so we can get rid of the re-export entirely.